.gitignore

```diff
@@ -55,3 +55,4 @@ apps/mantine.dev/out
 apps/mantine.dev/.next
 apps/mantine.dev/src/.docgen/
 .samples
+docs/
```

Comment:
Do not modify gitignore, remove this change.

Reply:
Ok, updated